test/Mvc6Framework45.FunctionalTests/Models/AccountViewModels.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.Rendering;
```

Comment:
If it compiles fine, that worries me *a lot*. That means there's a problem in the build system.

---
In reply to: [59235688](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/142#discussion_r59235688) [](ancestors = 59235688)